docs-chef-io/content/inspec/attestations.md

```diff
@@ -0,0 +1,95 @@
++++
+title = "Attestations"
+draft = false
+gh_repo = "inspec"
+
+[menu]
+  [menu.inspec]
+    title = "Attestations"
+    identifier = "inspec/reference/attestations.md Attestations"
+    parent = "inspec/reference"
+    weight = 140
++++
+
+Attestation is a mechanism to mark `Not Reviewed (N/R)` controls to `passed` or `failed` manually using an attestation file.An attestation file identifies:
+
+1. which controls needs to be attested.
+2. an explanation of why it is manually attested.
+3. control status `passed` or `failed` to attest controls.
+3. (optionally) an URL pointing to website containing information on control attestation.
+4. (optionally) an expiration date of attestation.
+
```

Comment:
We also need a note explaining how attestations are different that waivers - when you would use each.

Reply:
Should we include that in attestations document? In my opinion it would confuse the user about it. We already have a separate waivers doc explaining waivers mechanism. And this is another mechanism.

Reply:
(discussed)